public/video-ui/src/components/YoutubeFurniture/index.js

```diff
@@ -37,7 +37,7 @@ class YoutubeFurniture extends React.Component {
   }
 
   validateYouTubeDescription = description => {
-    return description.match(/<|>/)
+    return description && description.match(/<|>/)
```

Comment:
would the error message be correct in the case of an empty description?

Reply:
Yes, if there is an empty description we don't return an error and don't get an error message which is the desired behaviour.